index.js

```diff
@@ -43,13 +41,30 @@ module.exports = opts => {
     typeof handlers[type] === 'function' ? handlers[type] : missing(type)
 
   const processJob = ({ type, payload }, callback) => {
-
-    const done          = once(callback)
-          details       = mergeAll([ options, { type, payload }, timeoutErr ]),
-          logTimeout    = partial(timeoutLogger, [ details ]),
-          handleTimeout = compose(partial(done, [ new Error(timeoutErr.error) ]), tap(logTimeout))
-          timeout       = setTimeout(handleTimeout, visibilityTimeout * 1000),
-          finish        = compose(tap(partial(clearTimeout, [ timeout ])), done)
+    const done = once(callback)
+
+    const addErrorContext = error => {
+      error.params = mergeAll([ options, { type, payload }, timeoutErr ])
+      error.component = 'squiss-jobs'
+      return error
```

Comment:
This function is mutable, and can sometimes be called twice on the same `error`, which is kinda 👃 .